## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#b0fb3f2f-0893-420c-9a7a-8029d7d97e77) |
| mint-compression-fs.sh | :x: [more...](#600fbe22-8057-445d-8ab3-94adb7664d2e) |
<a name="b0fb3f2f-0893-420c-9a7a-8029d7d97e77"></a>
### 8416-bafbd44/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32104
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d425a1e0e9e4:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 9 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 1 minutes and 2 seconds
(4/14) Running aws-sdk-ruby tests ... done in 31 seconds
(5/14) Running awscli tests ... done in 1 minutes and 51 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 7 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 31 seconds
(9/14) Running minio-go tests ... FAILED in 4 minutes and 17 seconds
{
  "args": {},
  "duration": 80692,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```
<a name="600fbe22-8057-445d-8ab3-94adb7664d2e"></a>
### 8416-bafbd44/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:31740
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 1346e50254df:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 7 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 2 minutes and 0 seconds
(6/14) Running healthcheck tests ... done in 2 seconds
(7/14) Running mc tests ... done in 47 seconds
(8/14) Running minio-dotnet tests ... done in 32 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 16 seconds
(10/14) Running minio-java tests ... done in 3 minutes and 20 seconds
(11/14) Running minio-js tests ... done in 45 seconds
(12/14) Running minio-py tests ... FAILED in 5 minutes and 33 seconds
{
  "name": "minio-py:test_presigned_get_object_expiry_5sec",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1341, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 2036, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1355, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "message": "AccessDenied: message: Access Denied",
  "args": {
    "response._body": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>AccessDenied</Code><Message>Request has expired</Message><Key>a93f30b4-e438-400a-bbe6-6eae6e693f9b</Key><BucketName>minio-py-test-1256d7cd-0212-4891-a16e-5f1423d6dc14</BucketName><Resource>/minio-py-test-1256d7cd-0212-4891-a16e-5f1423d6dc14/a93f30b4-e438-400a-bbe6-6eae6e693f9b</Resource><RequestId>15D28AC0348E3ED6</RequestId><HostId>17c5c283-21ef-4d8f-806a-3f953221cefc</HostId></Error>",
    "bucket_name": "minio-py-test-1256d7cd-0212-4891-a16e-5f1423d6dc14",
    "presigned_get_object_url": "http://72.28.97.56:31740/minio-py-test-1256d7cd-0212-4891-a16e-5f1423d6dc14/a93f30b4-e438-400a-bbe6-6eae6e693f9b?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20191030%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20191030T214245Z&X-Amz-Expires=5&X-Amz-SignedHeaders=host&X-Amz-Signature=8514364648fdd8bd5eba171ff3ff23c6b7b93ef0269ecb4be2fd2b4641beae40",
    "response.status": 403,
    "response.reason": "Forbidden",
    "object_name": "a93f30b4-e438-400a-bbe6-6eae6e693f9b",
    "response.headers": "{\"_container\": {\"accept-ranges\": [\"Accept-Ranges\", \"bytes\"], \"content-length\": [\"Content-Length\", \"440\"], \"content-security-policy\": [\"Content-Security-Policy\", \"block-all-mixed-content\"], \"content-type\": [\"Content-Type\", \"application/xml\"], \"server\": [\"Server\", \"MinIO/DEVELOPMENT.2019-10-30T21-08-15Z\"], \"vary\": [\"Vary\", \"Origin\"], \"x-amz-request-id\": [\"X-Amz-Request-Id\", \"15D28AC0348E3ED6\"], \"x-xss-protection\": [\"X-Xss-Protection\", \"1; mode=block\"], \"date\": [\"Date\", \"Wed, 30 Oct 2019 21:42:58 GMT\"]}}"
  },
  "status": "FAIL",
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)",
  "duration": 95
}

Executed 11 out of 14 tests successfully.
```
Error: No such image: minio/minio:8416-bafbd44

--- a/cmd/daily-lifecycle-ops.go
+++ b/cmd/daily-lifecycle-ops.go
@@ -169,7 +169,7 @@ func lifecycleRound(ctx context.Context, objAPI ObjectLayer) error {
 						Object: ObjectInfo{
 							Name: objects[i],
 						},
-						Host: "[ILM-EXPIRY]",
+						Host: "Internal: [ILM-EXPIRY]",
 					})
 				}
 			}

